machine/accton/accton_as7412_32x/busybox/conf/config

```diff
@@ -0,0 +1,7 @@
+CONFIG_SYS_EEPROM=y
+CONFIG_SYS_EEPROM_DEVICE_I2C=y
+CONFIG_SYS_EEPROM_I2C_DEVICE="/dev/i2c-1"
+CONFIG_SYS_EEPROM_I2C_ADDR=0x57
+CONFIG_SYS_EEPROM_OFFSET=0
+CONFIG_SYS_EEPROM_SIZE=256
+CONFIG_SYS_EEPROM_MAX_SIZE=256
```

Comment:
I see all 3 of these new switches have 256 for the EEPROM size.  That is pretty small.  Is that correct?